schema/mysql/upgrades/1.0.0-rc2.sql

```diff
@@ -173,10 +174,17 @@ ALTER TABLE service_state
     MODIFY long_output longtext DEFAULT NULL,
     MODIFY performance_data longtext DEFAULT NULL;
 
+ALTER TABLE notification_history
+    MODIFY id binary(20) NOT NULL COMMENT 'sha1(environment.name + notification.name + send_time)';
```

Comment:
The change adding type is missing here.

Reply:
Don’t say this file still exists?

Reply:
At the moment, it's still there, and as long as it is, I'd still keep it up-to-date.

Reply:
... unless it's gone in the **next** moment.

Reply:
Come on, the change is trivial. At the moment, the PR still updates the upgrade file, but in a way that doesn't match the schema change. Do you mind if I just push a commit adding " + type" here too?